bin/oneoff/wipe_data/opt_out_petition_emails_under_16

```diff
@@ -13,11 +13,16 @@ require src_dir 'database'
 load pegasus_dir('helpers.rb')
 require 'cdo/only_one'
 
+# Twenty minutes.
+MAX_EXECUTION_TIME = 1_200_000
```

Comment:
Could we use a multiplication to generate this?  Otherwise a newcomer's first step is to probably do the multiplication by hand to make sure it's correct :)